theme_bistro/views/snippets/s_carousel.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_carousel" inherit_id="website.s_carousel">
+    <xpath expr="(//div[hasclass('container')])[2]" position="before">
+        <div class="o_we_bg_filter bg-white-50"/>
```

Comment:
😉

| this PR | proposal |
|--------|--------|
| <img width="1372" alt="image" src="https://github.com/user-attachments/assets/5baeca75-c422-456e-b506-0ed960120678"> | <img width="1370" alt="image" src="https://github.com/user-attachments/assets/d00bd192-2db5-4701-890b-d53c1c779cb8"> |